src/binary.ts

```diff
@@ -310,6 +325,213 @@ export class Binary extends BSONValue {
     const subTypeArg = inspect(this.sub_type, options);
     return `Binary.createFromBase64(${base64Arg}, ${subTypeArg})`;
   }
+
+  /**
+   * If this Binary represents a Int8 Vector,
+   * returns a copy of the bytes in a new Int8Array.
```

Comment:
I'd mention in the doc comments that these methods throw if they are not of the right subtype and ... idk, subsubtype or whatever we call it?